test/test-debuglet.js

```diff
@@ -243,6 +244,78 @@ describe('Debuglet', function() {
         assert.strictEqual(debuglet.config_.serviceContext.version,
                            'fake-gae-version');
       });
+
+      it('should respect GAE_MINOR_VERSION env. var. when available',
+         function() {
+           process.env.GAE_MINOR_VERSION = 'some minor version';
+           var debug = require('../src/debug.js')();
+           var debuglet = new Debuglet(debug, defaultConfig);
+           assert.ok(debuglet.config_);
+           assert.ok(debuglet.config_.serviceContext);
+           assert.strictEqual(debuglet.config_.serviceContext.minorVersion,
+                              'some minor version');
+         });
+
+      it('should conjure a fake minor version when running on flex',
+         function() {
+           process.env.GAE_SERVICE = 'fake-gae-service';
```

Comment:
Perhaps we should have tests that verify that if `GAE_MINOR_VERSION` is either the empty string or `undefined`, then the `GAE_SERVICE` value is analyzed.